test_trainer.py

```diff
@@ -140,8 +140,19 @@ def delete_on_exit(exp):
     parser = ArgumentParser()
     parser.add_argument("--no_delete", action="store_true", default=False)
     parser.add_argument("--no_end_to_end", action="store_true", default=False)
+    parser.add_argument("--include", "-i", nargs="+", default=[])
+    parser.add_argument("--exclude", "-e", nargs="+", default=[])
     args = parser.parse_args()
```

Comment:
add `include` and `exclude` args to select tests to run

To run the kitti test only:

```
python test_trainer.py --no_delete -i 9
```